### PR TITLE
Hopefully fix `renovate` for non-major versions & out of work hours frequency

### PR DESCRIPTION
**Pull request by paddyroddy**
Ideas came from https://www.augmentedmind.de/2023/07/30/renovate-bot-cheat-sheet

Found this setting 🤞 
image

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,11 +1,13 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base", ":disableDependencyDashboard", ":enablePreCommit"],
+  "extends": ["config:base", "schedule:nonOfficeHours", ":enablePreCommit"],
```
Comment (paddyroddy) [inline renovate.json:3]: Enable dependency dashboard so I can tell if we have configured it correctly

**Review by milanmlft — APPROVED** [review]
👍

**Comment by paddyroddy** [comment]
Ideas came from https://www.augmentedmind.de/2023/07/30/renovate-bot-cheat-sheet

**Comment by paddyroddy** [comment]
Encouraging... #161

**Comment by paddyroddy** [comment]
image
Presumably the actual merging happens [out of work hours](https://docs.renovatebot.com/presets-schedule/#scheduleautomergenonofficehours)

**Comment by samcunliffe** [comment]
🤞

**Comment by paddyroddy** [comment]
Nothing happened 😢

**Comment by paddyroddy** [comment]
Noticed a lot of branches are there
image

**Comment by paddyroddy** [comment]
[logs](https://developer.mend.io/github/UCL-ARC/python-tooling/-/job/018ad15e-8a7f-7ee5-884b-f7eab22292aa)

**Pull request merged**